packages/docusaurus-theme-classic/src/theme/BlogPostPage/StructuredData/index.tsx

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import {useBaseUrlUtils} from '@docusaurus/useBaseUrl';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import type {Props} from '@theme/BlogPostPage/StructuredData';
+
+export default function BlogPostStructuredData(props: Props): JSX.Element {
```

Comment:
FWIW there's slight duplication with `BlogListPage/StructuredData`. Don't have great refactoring suggestions though.

Reply:
Oh how do mean? I'm not sure I follow!

Reply:
I meant this file shares some logic with `BlogListPage/StructuredData/index.tsx`. I haven't checked how largely they coincide, though.

Reply:
Oh I see what you mean - yes; say the Person structured data object is the same in both I think and potentially shareable. But it's not massive and I also wasn't sure where it would make sense to live.

Reply:
I'd move some logic to theme-common in a `structuredDataUtils.ts` to avoid this kind of duplication.